packages/core/src/application.ts

```diff
@@ -51,6 +57,7 @@ export class Application extends Context {
     if (!options) options = {};
 
     this.bind('http.port').to(options.http ? options.http.port : 3000);
+    this.api({basePath: '/', paths: {}});
```

Comment:
Right now we are using Swagger/OpenAPI Spec 2.0, see #182 for upgrading to v3.0.

As for `app.api()` vs. `app.rest()` and `app.grpc()`, maybe we can stick with current `app.api()` and later, when we add more transports, we can change `app.api()` into a deprecated alias for `app.rest()`. 

Thoughts?